modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -106,6 +106,52 @@
                   "if": { "properties": { "homeless": { "const": false } } },
                   "then": { "required": ["address"] }
                 },
+                "claimant": {
+                  "type": "object",
+                  "properties": {
+                    "address":  {
+                      "type": "object",
+                      "properties": {
+                        "addressLine1":    { "type": "string", "maxLength": 60 },
+                        "addressLine2":    { "type": "string", "maxLength": 30 },
+                        "addressLine3":    { "type": "string", "maxLength": 10 },
+                        "city":            { "type": "string", "maxLength": 60 },
+                        "stateCode": {
```

Comment:
Some day this can become a `$ref` and we'll just have a collection of shared schemas!

Reply:
shared schemas ticket!